mmdet/datasets/custom.py

```diff
@@ -88,13 +88,25 @@ def __init__(self,
                 self.proposal_file = osp.join(self.data_root,
                                               self.proposal_file)
         # load annotations (and proposals)
-        with self.file_client.get_local_path(self.ann_file) as local_path:
-            self.data_infos = self.load_annotations(local_path)
+        if hasattr(self.file_client, 'get_local_path'):
+            with self.file_client.get_local_path(self.ann_file) as local_path:
+                self.data_infos = self.load_annotations(local_path)
+        else:
+            warnings.warn(
+                'We treat the self.ann_file as local paths and it might '
```

Comment:
Refine the warning message: f'The used MMCV version does not have get_local_path. We treat the {self.ann_file} as local paths and it might cause errors if the path is not a local path. Please use MMCV>= xxx if you meet errors'